### PR TITLE
croutonfreon.so: Disable cursor on switch away from Chromium OS

### PR DESCRIPTION
**Pull request by drinkcat**
For reasons that I do not fully comprehend, cursor sometimes get corrupted in crouton (see #2878, http://crbug.com/482801).

It does appears the resetting cursors on all CRTCs fixes the issue, so let's do that.

### CODE REVIEW

**Comment by drinkcat** [comment]
I'm not going to pretend I understand what's going on (and I don't really have the cycles to figure it out ,-( ). But, this seems to fix the issue.

**Comment by dnschneid** [comment]
Thanks for the black magic!  Does libdrm-dev need to be installed for compilation?

**Comment by drinkcat** [comment]
2016-11-24_07-53-46_drinkcat_chroagh_freon_cursor/status.html ... Turns out libdrm header files appear to be broken on most releases... Looking.

File: src/freon.c [hunk 1]
```diff
@@ -95,6 +97,38 @@ static int set_display_lock(unsigned int pid) {
...
+    drmModeRes* resources;
+
+    fd = open("/dev/dri/card0", O_RDWR, 0);
+
+    TRACE("%s %d\n", __func__, fd);
+
+    if (fd < 0)
+        return;
+
+    resources = drmModeGetResources(fd);
```
Comment (drinkcat) [inline src/freon.c]: Done

**Comment by drinkcat** [comment]
Manually tested on precise/trusty/xenial and stretch (all on samus). I'll launch a test as well.

**Comment by drinkcat** [comment]
Mostly pass `2016-11-25_02-51-07_drinkcat_chroagh_freon_cursor_35/status.html`. `sid` is broken, but I don't think that's related. Also, x86-alex is weird, not sure what's going on but does not seem related either.

**Pull request merged**